tests/Unit/PointwiseFunctions/AnalyticSolutions/Xcts/Test_Schwarzschild.cpp

```diff
@@ -0,0 +1,127 @@
+#include "Utilities/TaggedTuple.hpp"
+
+namespace {
+
+using field_tags =
+    tmpl::list<Xcts::Tags::ConformalFactor<DataVector>,
+               Xcts::Tags::LapseTimesConformalFactor<DataVector>,
+               Xcts::Tags::ShiftExcess<DataVector, 3, Frame::Inertial>>;
+using auxiliary_field_tags =
+    tmpl::list<::Tags::deriv<Xcts::Tags::ConformalFactor<DataVector>,
+                             tmpl::size_t<3>, Frame::Inertial>,
+               ::Tags::deriv<Xcts::Tags::LapseTimesConformalFactor<DataVector>,
+                             tmpl::size_t<3>, Frame::Inertial>,
+               Xcts::Tags::ShiftStrain<DataVector, 3, Frame::Inertial>>;
+using background_tags =
+    tmpl::list<Xcts::Tags::ConformalMetric<DataVector, 3, Frame::Inertial>,
+               gr::Tags::TraceExtrinsicCurvature<DataVector>,
+               ::Tags::deriv<gr::Tags::TraceExtrinsicCurvature<DataVector>,
+                             tmpl::size_t<3>, Frame::Inertial>,
+               Xcts::Tags::ShiftBackground<DataVector, 3, Frame::Inertial>>;
+using matter_source_tags =
+    tmpl::list<gr::Tags::EnergyDensity<DataVector>,
+               gr::Tags::StressTrace<DataVector>,
+               gr::Tags::MomentumDensity<3, Frame::Inertial, DataVector>>;
+using fixed_source_tags = db::wrap_tags_in<Tags::FixedSource, field_tags>;
+
+template <Xcts::Solutions::SchwarzschildCoordinates Coords>
+struct SchwarzschildProxy : Xcts::Solutions::Schwarzschild<Coords> {
+  using Xcts::Solutions::Schwarzschild<Coords>::Schwarzschild;
+  tuples::tagged_tuple_from_typelist<
+      tmpl::append<field_tags, auxiliary_field_tags>>
+  field_variables(const tnsr::I<DataVector, 3, Frame::Inertial>& x) const
+      noexcept {
+    return Xcts::Solutions::Schwarzschild<Coords>::variables(
+        x, tmpl::append<field_tags, auxiliary_field_tags>{});
+  }
+  tuples::tagged_tuple_from_typelist<background_tags> background_variables(
+      const tnsr::I<DataVector, 3, Frame::Inertial>& x) const noexcept {
+    return Xcts::Solutions::Schwarzschild<Coords>::variables(x,
+                                                             background_tags{});
+  }
+  tuples::tagged_tuple_from_typelist<matter_source_tags>
+  matter_source_variables(
+      const tnsr::I<DataVector, 3, Frame::Inertial>& x) const noexcept {
+    return Xcts::Solutions::Schwarzschild<Coords>::variables(
+        x, matter_source_tags{});
+  }
+  tuples::tagged_tuple_from_typelist<fixed_source_tags> fixed_source_variables(
+      const tnsr::I<DataVector, 3, Frame::Inertial>& x) const noexcept {
+    return Xcts::Solutions::Schwarzschild<Coords>::variables(
+        x, fixed_source_tags{});
+  }
+};
+
+template <Xcts::Solutions::SchwarzschildCoordinates Coords>
+void test_solution(const double radius_at_horizon,
+                   const std::string& py_functions_suffix) {
+  CAPTURE(Coords);
+  const SchwarzschildProxy<Coords> solution{};
+  CHECK(solution.radius_at_horizon() == approx(radius_at_horizon));
```

Comment:
I'd bump this to a `REQUIRE` since the rest of the test doesn't make sense (i.e. the domain is possibly wrong) if this isn't true